jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/TextArea.java

```diff
@@ -37,9 +37,10 @@ public void setLines(String... lines) {
     public int maxlength() { return getInt("maxlength", uiElement); }
     @JDIAction("Add ne line '{0}' in '{name}'")
     public void addNewLine(String line) {
+        String nLine = null;
         if (isNotEmpty())
-            line = "\n" + line;
-        sendKeys(line);
+            nLine = "\n" + line;
+        sendKeys(nLine);
```

Comment:
If TextArea is empty, nLine will be null, and this string will throw an exception.

Reply:
Few ways to avoid exception:
```
    public void addNewLine(String line) {
        String nLine = line;
        if (isNotEmpty()) {
            nLine = "\n" + nLine;
        }
        sendKeys(nLine);
    }
```
```
 public void addNewLine(String line) {
     sendKeys(isNotEmpty() ? "\n" + line : line );
}
```